core/rawdb/schema.go

```diff
@@ -58,18 +58,18 @@ var (
 	shardStatePrefix = []byte("ss") // shardStatePrefix + num (uint64 big endian) + hash -> shardState
 	lastCommitsKey   = []byte("LastCommits")
 
+	lastPendingCrosslinkKey = []byte("lastPendingCL") // prefix for shard last pending crosslink
```

Comment:
Still, why call last Pending? pending is enough.

Reply:
remove "last"